calico_containers/tests/unit/datastore_test.py

```diff
@@ -126,6 +126,74 @@ def test_pprint(self):
                      rule4.pprint())
 
 
+class TestRules(unittest.TestCase):
+
+    def test_rules(self):
+        """
+        Create a detailed set of rules, convert from and to json and compare
+        the results.
+        """
+        rules_json = """
+{
```

Comment:
Does the json converter care about indentation? It'd be cool to indent this as normal, so that code folding et al. works.